### ISSUE TITLE
Assigned task to a new member faild

### ISSUE THREAD

**Issue by Grujko** [gregoryagu/IssueTracker#72]
Environment: Windows 10 pro
Browser: Google Chrome

Reproduction steps:
1) Log in on the page.
2) Click on the "Explorer" button.
3) Click on the "+" icon to add new Task.
5) In the "Name" field, submit task name. Task name: Task for Mile
5) Check "Task" box.
6) Click on the "Save" button.

Expected results:
User should be able to create and assign new task to a new team member.

Actual results:
User is not able to assign created task to a new member.
**User can assign task to a new member only after he finished creating of a new task, and after that  User must open Edit function of the wanted task to assign them to a new member.**

Attachment: video link 
https://screencast-o-matic.com/watch/cqVnlPrSiR

**Comment by gregoryagu** [comment]
Thank you very much for this.

I have updated the behavior so that it works more like you would expect.  After the task is created, it is no longer in Edit Mode.  

So now after creating the task, just click edit mode and the details of the task can be edited.

**Comment by nikhilgkulkarni1** [comment]
Verified.
